src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
The `get()` method should be properly parameterized with the generic type parameter `T` instead of using a wildcard. This ensures that the method returns a list of a specific subtype of `Machine`. For example, it should be `List<T> get();`.

Reply:
The get() method should be properly parameterized with the generic type parameter T